README.rst

```diff
@@ -9,7 +9,32 @@ Crate Commoncrawl Importer Plugin
 =================================
 
 This plugin is used to fast import data from `Common Crawl`_ into
-Crate_.
+Crate_. The plugin extends a custom schema ``ccrawl`` to the copy 
+from command which allows to import gzipped common crawl data to
+a table.
+
+The table should have the following schema::
+
+    create table if not exists commoncrawl (
+      ssl boolean primary key, -- http/https - byte/boolean primary key
+      authority string primary key, -- xyz.hello.com:123 primary key
+      path string primary key, -- /a?d=1#hello primary key
+      date timestamp primary key,
+      week_partition as date_trunc('week', date) primary key,
+      ctype string,
+      clen int,
+      content string INDEX using fulltext with (max_token_length = 40)
+    ) clustered into 8 shards;
```

Comment:
the amount of shards is just something I came up with. in reality we used 200 :)